cheat_sheets/pyfluent_cheat_sheet/pyfluent_cheat_sheet.tex

```diff
@@ -44,8 +44,9 @@ \section{\includegraphics[height=\fontcharht\font`\S]{slash.png} Launch Fluent l
 This code starts Fluent in the background so that commands can be sent to Fluent from the Python interpreter.
 
 \begin{lstlisting}[language=Python]
-import ansys.fluent.core as pyfluent
-solver = pyfluent.launch_fluent(mode = "solver", show_gui = True)
+  import ansys.fluent.core as pyfluent
```

Comment:
```suggestion
import ansys.fluent.core as pyfluent
```
Do we require this indent before? is it a suggestion from review?

Reply:
That indendt shouldn't be there

Reply:
Removed all the indent!